### PR TITLE
Update supermodel.rb

### PR DESCRIPTION
**Pull request by lourinaldi**
HEAD version has renamed Makefile

- [X ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by issyl0 — CHANGES_REQUESTED** [review]
Hi @lourinaldi and thanks for this PR! 💪

CI is failing because it's trying to find the renamed `Makefile.OSX` in the currently released version. CI doesn't build HEAD of any formulae, as far as I know.

```
Errno::ENOENT: No such file or directory @ rb_sysopen - Makefiles/Makefile.OSX
```

The commit message says that it's changed in HEAD. Therefore, this PR needs to leave the main `install` block as it is, so that non-HEAD users can build from source with the last release's Makefile paths. You can make this change apply to only HEAD builds with an `if build.head?` conditional.

To avoid duplication, it would be good to assign the Makefile path to a variable depending on whether it's a HEAD build or not, such as:

```ruby
makefile_dir = build.head? ? Makefiles/Makefile.OSX : Makefiles/Makefile.SDL.OSX.GCC
```

and then use that `makefile_dir` variable in both the places that you've changed here. Assuming all the `inreplace` lines still apply cleanly! :grimacing:

Alternatively, do you know if the software author plans to release a new version soon given these changes? It would be good to change this for everyone at once if there is a new version planned soon.

**Comment by lourinaldi** [comment]
I'll defer to @trzy on your last question. It doesn't look like he has cut a release in over 8 years(!!)

File: Formula/supermodel.rb
Comment (issyl0) [inline Formula/supermodel.rb]:
```suggestion
    makefile_dir = build.head? ? "Makefiles/Makefile.OSX" : "Makefiles/Makefile.SDL.OSX.GCC"
```

Comment (issyl0) [reply Formula/supermodel.rb]: Oops, I forgot the quotes in my example. :flushed: Please accept those changes, or tick the "allow maintainers to push" box in the corner of this PR, then this should pass CI.

**Review by issyl0 — APPROVED** [review]
This looks good now. Thanks for your first contribution to Homebrew!

**Pull request merged**
